### PR TITLE
Upgrade dom-testing-library to 5.0

### PR DESCRIPTION
**Pull request by jihchi**
As title.

### CODE REVIEW

**Comment by jihchi** [comment]
Hi @wyze ,

Could you please take a look when you have time?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-dom-testing-library",
-  "version": "0.4.1",
+  "version": "0.5.0",
```
Comment (wyze) [inline package.json]: Revert this as it is done when published to npm.

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## Change Log
 
+### [v0.5.0](https://github.com/wyze/bs-dom-testing-library/releases/tag/v0.5.0) (2019-07-05)
```
Comment (wyze) [inline changelog.md]: Revert this as it is done when published to npm.

**Review by wyze — CHANGES_REQUESTED** [review]

**Comment by wyze** [comment]
Sorry for the delay, was on vacation and got back and had to catch up on a ton of stuff!

File: package.json (hunk 1 above)
Comment (jihchi) [reply package.json]: Done.

File: changelog.md (hunk 2 above)
Comment (jihchi) [reply changelog.md]: Done.

**Comment by jihchi** [comment]
No worries! I've reverted and pushed the commit.

**Review by wyze — APPROVED** [review]

**Pull request merged**